recipe/meta.yaml

```diff
@@ -11,48 +11,44 @@ source:
   sha256: d1877192b44d36055f39fe212ca0abf04416bac1ff81be1b6cda0fbe82c12ac5
 
 build:
-  number: 1
+  number: 2
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m build -C--global-option=--release
```

Comment:
looks like we can add a test command `pip list | grep nncf` to check that NNCF is still visible by pip after this change

Reply:
I had to roll this back anyway.